git/reflog.go

```diff
@@ -0,0 +1,6 @@
+package git
+
+// Returns true if a reflog exists for refname r under client.
+func ReflogExists(c *Client, r Refname) bool {
+	return c.GitDir.File(File(r)).Exists()
```

Comment:
I thought that the ref logs are the files that are typically in .git/logs/refs/... that store a log of the changes to the ref. Shouldn't this check if c.GitDir.File(File("logs")).File(File(r)).Exists()?

Reply:
oops.. you're right.

Reply:
should be fixed now